### DISCUSSION TITLE
App level keyboard event/hot key

### DISCUSSION THREAD

**Discussion by crackleeeessyp** [tauri-apps/tauri#11828] [Q&A]
We can use Global Shortcut plugin for global shortcut, however, it is better to have app-level keybaord/hot key support in rust. 

related discussion:

https://github.com/tauri-apps/tauri/issues/11671

**Comment by FabianLars** [comment]
Sorry but what is the point of this discussion thread? You already linked the tracking issue so i'm quite confused tbh
